molecule/default/tests/test_default.py

```diff
@@ -1,20 +1,60 @@
 import os
 import re
-
+import pytest
 import testinfra.utils.ansible_runner
 
+###############################################################
+# ASC-187: Verify nova_force_config_drive setting is disabled  #
+################################################################
+# RPC 10+ manual test 1
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('compute-infra_hosts')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts(
+    ['nova_scheduler', 'nova_compute'])
 
 
-# RPC 10+ manual test 1
-def test_nova_force_config_drive_is_disabled(host):
+def test_nova_force_config_drive_is_disabled_on_nova_compute(host):
+    verify_nova_force_config_drive_is_disabled(host)
+
+
+def test_nova_force_config_drive_is_disabled_on_nova_scheduler(host):
+    verify_nova_force_config_drive_is_disabled(host)
+
+
+def verify_nova_force_config_drive_is_disabled(checked_host):
```

Comment:
What is `checked_host`? I don't see it declared.

Reply:
Molecule won't consider this function as a test (since it doesn't start with `test_*`, I use this as a library function called by test functions